sct.py

```diff
@@ -1951,6 +1953,123 @@ def upload_artifact_file(test_id: str, file_path: str, use_argus: bool, public:
         return
 
 
+@cli.command("hdr-investigate", help="Analyze HDR file for latency spikes")
+@click.option("--test-id", type=str, required=False, help="If hdr_folder is not provided, logs will be downloaded from argus "
+                                                          "using this test_id")
+@click.option("--stress-tool", type=str, default="cassandra-stress", required=False,
```

Comment:
set the options available, use click method of doing so.

Reply:
changed